Library/Formula/ndenv.rb

```diff
@@ -0,0 +1,30 @@
+class Ndenv < Formula
+  homepage "https://github.com/riywo/ndenv"
+  url "https://github.com/riywo/ndenv/archive/v0.4.0.tar.gz"
+  sha1 "f07199b2dd4e28ef9e63e1a9cf17f71c874a1c61"
+
+  def install
+    inreplace "libexec/ndenv", "/usr/local", HOMEBREW_PREFIX
+    prefix.install Dir["*"]
+
+    #  Create a plugin and link it into ndenv's opt_prefix. Only create if it
+    #    does not already exist (user might have installed stuff manually).
+    plugins_path = Pathname.new "#{HOMEBREW_PREFIX}/share/ndenv/plugins"
```

Comment:
`plugins_path = HOMEBREW_PREFIX/"share/ndenv/plugins"` creates a Pathname object. Can you do that below too? Thanks.